sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/PreDefinedTag.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.ResourceManager.Core
+{
+    /// <summary>
+    /// A class representing a Tags along with the instance operations that can be performed on it.
+    /// </summary>
+    public class PreDefinedTag : PreDefinedTagsOperations
```

Comment:
inconsistent plural here.  I believe the resource is
PreDefinedTag
So the operations object should be PreDefinedTagOperations

Reply:
Done